rdr_service/genomic/genomic_job_components.py

```diff
@@ -1738,6 +1743,32 @@ def aw5_array_manifest_name_rule():
 
         return is_valid_filename
 
+    def validate_values(self, data):
+        is_invalid, message = False, None
+        cleaned_fieldnames = [self._clean_field_name(fieldname) for fieldname in data['fieldnames']]
+
+        values_to_check = {
```

Comment:
Can this dict be defined on the manifest level and then pulled into this method?